### PR TITLE
Api endpoint for poy v0.5

### PR DESCRIPTION
**Pull request by HrikB**
The new API endpoint provides all the data for the first page of proof of yield. 
image

### CODE REVIEW

File: eagleproject/core/blockchain/addresses.py [hunk 1]
```diff
@@ -144,6 +144,7 @@
 METAMASK_SWAP_ROUTER = "0x881d40237659c251811cec9c364ef91dc08d300c"
 FLIPPER = "0xcecad69d7d4ed6d52efcfa028af8732f27e08f70"
 DRIPPER = "0x80c898ae5e56f888365e235ceb8cea3eb726cb58"
+OETH_DRIPPER = "0xc0F42F73b8f01849a2DD99753524d4ba14317EB3"
```
Comment (shahthepro) [inline eagleproject/core/blockchain/addresses.py:147]: This should be in lower case. Checksum addresses don't work on this repo

File: eagleproject/core/blockchain/rpc.py [hunk 2]
```diff
@@ -463,10 +464,15 @@ def staking_durationRewardRate(address, duration, block="latest"):
...
     signature = SIG_DRIPPER_AVAILABLE_FUNDS[:10]
-    data = call(DRIPPER, signature, "", block)
-    return Decimal(int(data["result"], 16)) / E_6
+
+    if project == OriginTokens.OETH:
+        data = call(OETH_DRIPPER, signature, "", block)
+        return Decimal(int(data["result"], 16)) / E_18
+    else:
+        data = call(DRIPPER, signature, "", block)
+        return Decimal(int(data["result"], 16)) / E_6
```
Comment (shahthepro) [inline eagleproject/core/blockchain/rpc.py:475]: OUSD dropper is still returning data in 6 decimals?

**Review by shahthepro — CHANGES_REQUESTED** [review]

File: eagleproject/core/blockchain/rpc.py (hunk 2 above)
Comment (HrikB) [reply eagleproject/core/blockchain/rpc.py:475]: I believe this is correct no? 
https://etherscan.io/address/0x80c898ae5e56f888365e235ceb8cea3eb726cb58#readProxyContract

image
image

Comment (shahthepro) [reply eagleproject/core/blockchain/rpc.py:475]: Yeah, just checked it against deployed OUSD Dripper and the codebase, seems to be right

**Review by shahthepro — APPROVED** [review]

**Pull request merged**